src/api/java/com/minecolonies/api/configuration/Configurations.java

```diff
@@ -113,6 +113,12 @@
         @Config.Comment("The minimum number of nights between raids")
         public int minimumNumberOfNightsBetweenRaids = 1;
 
+        @Config.Comment("Should Mobs attack citizens? default:false")
+        public boolean mobAttackCitizens = true;
```

Comment:
default false, but here it is true? =D

Reply:
oh forgot to change it back from the testing version :D